bin/pre-commit

```diff
@@ -2,8 +2,9 @@
 
 require 'pre-commit/cli'
 
-if !File.exists?(".git")
-  abort "No .git directory found."
-end
+# Change directory to the root of the git repository.
+repo_root = `git rev-parse --show-toplevel`.strip
+abort "No .git directory found." unless File.directory?(repo_root)
```

Comment:
I believe there's no guarantee that there is a `.git` folder at the root of everybody's repository, as it is configurable. See http://stackoverflow.com/a/505517.